README.md

```diff
@@ -80,10 +80,11 @@ You can [download](https://github.com/IIvexII/LGU-TimetableAPI/releases) the lat
 
```

Comment:
![image](https://user-images.githubusercontent.com/41378765/202946784-8ecf8f75-9a62-45a8-a6db-7597ddbedd47.png)

Can you please also change the text as seen in the screenshot and Credits to Technologies.

Reply:
No problem! I've just done it.